packages/core/src/components/Badge/Badge.example.jsx

```diff
@@ -5,12 +5,13 @@ import ReactDOM from 'react-dom';
 ReactDOM.render(
   <Fragment>
     <Badge>Default badge</Badge>
-    <Badge variation="info">Variant badge</Badge>
+    <Badge variation="info">Info badge</Badge>
     <Badge variation="success">
-      Badge with <b>nested</b>
+      Success Badge with <b>bold text</b>
```

Comment:
Hm, the only thought I have is that this is supposed to demonstrate an example with more complex `children` or multiple nested elements. Whether its bold or something else doesnt actually matter

Reply:
Ah, I see what you are saying. Makes sense to leave it as is